testproject-tools-integration/Packages/manifest.json

```diff
@@ -2,7 +2,7 @@
     "registry": "https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-candidates",
     "dependencies": {
         "com.unity.ide.rider": "3.0.7",
-        "com.unity.multiplayer.tools": "1.0.0-pre.5",
+        "com.unity.multiplayer.tools": "https://github.com/Unity-Technologies/com.unity.multiplayer.tools.git#release/1.0.0-pre.6",
```

Comment:
The reason for pointing to github is because we don't want to publish a package everytime we want to test.